x-pack/legacy/plugins/actions/README.md

```diff
@@ -35,6 +35,12 @@ Built-In-Actions are configured using the _xpack.actions_ namespoace under _kiba
 | _xpack.actions._**enabled**          | Feature toggle which enabled Actions in Kibana. Currently defaulted to false while Actions are experimental.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | boolean               |
 | _xpack.actions._**WhitelistedHosts** | Which _hostnames_ are whitelisted for the Built-In-Action? This list should contain hostnames of every external service you wish to interact with using Webhooks, Email or any other built in Action. Note that you may use the string "\*" in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential use of such a feature to execute [SSRF](https://www.owasp.org/index.php/Server_Side_Request_Forgery) attacks from your server. | Array<String> |
```

Comment:
we need tot fix this capitalized `W`, change it to `w` - it's been copied into tons of other issues, emails, etc.

Reply:
woah, didn't realize the line was so long - the config key ends up rendering as `xpack.actions.WhitetlistedHosts` when the actual key name is `xpack.actions.whitetlistedHosts`

Reply:
lol, good thing I didn't copy paste that... `whitelistedHosts`, not `whitetlistedHosts` :)